src/MOIWrapper.jl

```diff
@@ -65,6 +76,28 @@ function Optimizer(;kwargs...)
     return model
 end
 
+"""
+    _require_update(model::Optimizer)
+
+Sets the `model.needs_update` flag. Call this at the end of any mutating method.
+"""
+function _require_update(model::Optimizer)
+    model.needs_update = true
+    return
+end
+
+"""
+    _require_update(model::Optimizer)
+
+Calls `update_model!`, but only if the `model.needs_update` flag is set.
+"""
+function _update_if_necessary(model::Optimizer)
+    if model.needs_update
+        update_model!(model.inner)
+        model.needs_update = false
```

Comment:
It will return false if `model.needs_update` and `nothing` otherwise if I am not mistaken. Maybe add an explicit return ?